OWNERS

```diff
@@ -0,0 +1,49 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+approvers:
```

Comment:
Why not keep the original term committer?

Reply:
It's talked in https://internals.tidb.io/t/topic/854.

Reply:
Current it should not merged before all the other jobs be done, I will add a hold label to block it to merge, but we can give it a `LGTM` and `/merge` when we reach an agreement.

Reply:
+1, `commiter` is a more common term.

Reply:
It's a struct field, we could consider it equal our `commiter`.
https://www.kubernetes.dev/docs/guide/owners/#owners

<img width="519" alt="image" src="https://user-images.githubusercontent.com/2574558/230822177-ae6a40d0-e267-476d-8c1d-389cff857df2.png">